src/components/sync-dialog.tsx

```diff
@@ -0,0 +1,214 @@
+import { createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { useState } from 'react';
+import Button from 'src/components/button';
+import { RightArrowIcon } from 'src/components/icons/right-arrow';
+import Modal from 'src/components/modal';
+import { TreeView, TreeNode } from 'src/components/tree-view';
+import { useI18nData } from 'src/hooks/use-i18n-data';
+import { getIpcApi } from 'src/lib/get-ipc-api';
+import { getLocalizedLink } from 'src/lib/get-localized-link';
+import { CircleProdIcon } from './icons/circle-prod';
+import { CircleStagingIcon } from './icons/circle-staging';
+import type { SyncSite } from 'src/hooks/use-fetch-wpcom-sites/types';
+
+const allCopy = {
+	pull: {
+		staging: {
+			title: __( 'Pull from Staging' ),
+			description: __(
+				"Pulling will replace your Studio site's files and database with a copy from your staging site."
+			),
+		},
+		production: {
+			title: __( 'Pull from Production' ),
+			description: __(
+				"Pulling will replace your Studio site's files and database with a copy from your production site."
+			),
+		},
+		fromLabel: __( 'Pull' ),
+		toLabel: __( 'To' ),
+		subtitleSelector: __( 'What would you like to pull?' ),
+		envSync: __( 'Read more about <a>environment pull</a> ↗' ),
```

Comment:
Let's move the arrow inside the link.

```suggestion
		envSync: __( 'Read more about <a>environment pull ↗</a>' ),
```

Reply:
This will be fixed once https://github.com/Automattic/studio/pull/1474 is merged.